src/SamplesApp/UITests.Shared/Windows_UI_Xaml/Clip/XamlButtonWithClipping.xaml

```diff
@@ -1,36 +1,45 @@
-<UserControl
-	x:Class="Uno.UI.Samples.Content.UITests.Clip.XamlButtonWithClipping" 
-	xmlns:controls="using:Uno.UI.Samples.Controls"
-	xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
-	xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-	xmlns:local="Uno.UI.Samples.Content.UITests.Clip"
-	xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
-	xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
-	xmlns:u="using:Uno.UI.Samples.Controls"
-	xmlns:ios="http://uno.ui/ios"
-	xmlns:win="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
-	xmlns:android="http://uno.ui/android"
-	mc:Ignorable="d ios android"
-	d:DesignHeight="400"
-	d:DesignWidth="400">
-	
+<UserControl x:Class="Uno.UI.Samples.Content.UITests.Clip.XamlButtonWithClipping"
```

Comment:
The d using in some rare case can cause crashes while editing xaml file even when the app is not running. I usually remove it from any pages I ever work on since it's just useless if you are not using the design view.